components/bootloader/subproject/main/bootloader_start.c

```diff
@@ -87,14 +94,25 @@ static int selected_boot_partition(const bootloader_state_t *bs)
 #if CONFIG_BOOTLOADER_FACTORY_RESET_PIN_HIGH
         reset_level = true;
 #endif
+
+#ifdef CONFIG_BOOTLOADER_WAIT_AFTER_EN
+        ESP_LOGW(TAG, "Use GPIO %d for more than %d s to perform factory reset!",
```

Comment:
Does this message really need to be a warning? I think it would make more sense to have it as an `INFO`.

Reply:
judging by the fact the there is a "otherwise unnecessary" waiting time introduced, that substantially slows down the boot process (see review comment above about the 3 sec), I'd say that it should always be present, to remind to think about probably removing / resolving it..
But I get the reasoning behind it, Warnings make worry..
I dont have a problem lowering it to Info.. Just thought it shouldnt be "invisible", which it would be once you set the overall Log level to Warning..